bumble/controller.py

```diff
@@ -159,13 +159,13 @@ def __init__(
         self.total_num_le_acl_data_packets = 64
         self.iso_data_packet_length = 960
         self.total_num_iso_data_packets = 64
-        self.event_mask = 0
-        self.event_mask_page_2 = 0
+        self.event_mask = b'\x00'
+        self.event_mask_page_2 = b'\x00'
```

Comment:
Event masks are bit masks, so it's better to store them as integers. We can convert command parameters as integers to resolve this.

Reply:
How does it fail?